app/core/abilities/abilities.ts

```diff
@@ -272,16 +273,19 @@ export class PaydayStrategy extends AbilityStrategy {
     crit: boolean
   ) {
     super.process(pokemon, state, board, target, crit)
+    const factor = 0.5
     const damage = Math.floor(
-      ([30, 60, 90][pokemon.stars - 1] ?? 90) * (1 + (0.5 * pokemon.ap) / 100)
+      ([30, 60, 90][pokemon.stars - 1] ?? 90) *
+      (1 + (factor * pokemon.ap) / 100) *
+      (crit ? 1 + (pokemon.critPower - 1) * factor : 1)
     )
 
     const { death } = target.handleSpecialDamage(
       damage,
       board,
       AttackType.SPECIAL,
       pokemon,
-      crit,
+      false,
```

Comment:
That prevents crit effects like Rocky helmet to be handled correctly. The crit on damage is fine as it is